keyboards/montsinger/palmetto/mcuconf.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 Rossman360 (@rossmontsinger)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef RP_I2C_USE_I2C0
+#undef RP_I2C_USE_I2C1
+#define RP_I2C_USE_I2C0 FALSE
+#define RP_I2C_USE_I2C1 FALSE
```

Comment:
Since you've disabled I2C these shouldn't need to be changed, and this file can be deleted.